scene/3d/camera.cpp

```diff
@@ -201,15 +201,18 @@ void Camera::make_current() {
 	//get_scene()->call_group(SceneMainLoop::GROUP_CALL_REALTIME,camera_group,"_camera_make_current",this);
 }
 
-void Camera::clear_current() {
+void Camera::clear_current(bool enable_next) {
```

Comment:
Please name parameters with a `p_` prefix, so `p_enable_next` (also in the header, but not in the binding).